### ISSUE TITLE
Suggest splitting screen vertically

### ISSUE THREAD

**Issue by Pochtli137** [MisterPhilip/omnibug#127]
**Is your feature request related to a problem? Please describe.**
A lot of opening, scrolling and closing  when switching between data points.

**Describe the solution you'd like**
Make splitting the screen ala observe point (which you are better than in every way) a UI choice

![image](https://user-images.githubusercontent.com/72602823/95566300-a8601600-0a21-11eb-974b-f8caa628c194.png)

**Comment by MisterPhilip** [comment]
Yeah, this is something I've been thinking about for a while. Let me see what we can do in v2 for it!

**Comment by Pochtli137** [comment]
💪🤓

**Comment by Pochtli137** [comment]
Ok! Happy to lend a hand with design feedback and user testing if u need it.
